tests/python-pytest/onnx/test_operators.py

```diff
@@ -91,9 +91,11 @@ def test_onnx_export_zeros_like(tmp_path):
 
 
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
-def test_onnx_export_arange_like(tmp_path, dtype):
-    M = def_model('contrib.arange_like')
-    x = mx.nd.array([[-2,-1,0],[0,50,99],[4,5,6],[7,8,9]], dtype=dtype)
+@pytest.mark.parametrize("axis", [0,1,None])
+@pytest.mark.parametrize("test_data", [[[[0,1,2],[3,4,5]],[[6,7,8],[9,10,11]]], [[1,3],[2,4]]])
+def test_onnx_export_arange_like(tmp_path, dtype, axis, test_data):
```

Comment:
Can you also check the step option in the tests?

Reply:
Added.